packages/connect-iframe/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "./libDev",
-        "types": ["web", "w3c-web-usb", "jest"]
+        "types": ["web", "w3c-web-usb", "jest"],
+        "isolatedModules": false
```

Comment:
is this change really needed? I just tried running it locally without this and it wokrs.